### DISCUSSION TITLE
Robotic Additive manufacturing processes

### DISCUSSION THREAD

**Discussion by yvanblanchard** [OpenCommissioning/OC_TwinCAT_Core#3] [General]
Interested by having feedback about industrial needs related to virtual commissioning in the field of robotic additive manufacturing (could be Filament Winding, FDM/FGF Printing, Fiber/Tape placement) !

**Comment by mika-engineering** [comment]
We believe that regardless of the industry you are in, virtual commissioning is a great benefit for many reasons. We've been creating virtual models with various tools for 15 years now. That's where the open commissioning initiative comes from. I remember a few years ago we created a virtual model for a carbon fiber manufacturer, mainly because the customer wanted to test different placement recipes without wasting material.
